docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management-Transition.rst

```diff
@@ -1,6 +1,21 @@
 Object Lifecycle Management: Transition
 =======================================
 
+Object lifecycle transition policies enable you to change the location of an
+object or object type based on its age.
+
+Transition policies do not fully transition non-versioned sources. For such 
+sources, the object is copied to the destination, but it is not removed from 
+the source. For example, if a user sets a policy to transition an object from
+Azure (source) to AWS S3, the object is copied from Azure to AWS S3, but not
+removed from Azure.
+
+This affects the following location constraints: Azure, GCP, and--if the 
```

Comment:
Once we have this mechanism in place, GCP will be able to be versioned as well as non-versioned. Zenko does not yet support versioning on top of GCP, but the GCP bucket itself can be versioned, and that is what matters for supporting deletion in transition policies.

So I would simply move "GCP" to the "if the source is non-versioned" list.

It might be worth adding that sproxyd-RING is supported, regardless of the zenko bucket being versioned or nonversioned (same for S3 local storage aka. s3-data, but AFAIK we don't officially support it so not sure it should be documented).

Reply:
done